packages/next/src/withPayload.js

```diff
@@ -129,45 +118,14 @@ export const withPayload = (nextConfig = {}, options = {}) => {
 
       return {
         ...incomingWebpackConfig,
-        externals: [
-          ...(incomingWebpackConfig?.externals || []),
-          'drizzle-kit',
-          'drizzle-kit/api',
-          'sharp',
-          'libsql',
-          'require-in-the-middle',
-        ],
-        ignoreWarnings: [
-          ...(incomingWebpackConfig?.ignoreWarnings || []),
-          { module: /node_modules\/mongodb\/lib\/utils\.js/ },
-          { file: /node_modules\/mongodb\/lib\/utils\.js/ },
-          { module: /node_modules\/mongodb\/lib\/bson\.js/ },
-          { file: /node_modules\/mongodb\/lib\/bson\.js/ },
-        ],
+        externals: [...(incomingWebpackConfig?.externals || []), 'require-in-the-middle'],
```

Comment:
the dependencies in webpack.externals are now always externalized server-side - see serverExternalDependencies.

On the client, those dependencies are (and should) never imported anyways, thus no point in manually declaring them here